app/jobs/verify_linkbacks_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class VerifyLinkbacksJob < ApplicationJob
+  queue_as :default
+
+  retry_on ActiveRecord::RecordNotFound, wait: :exponentially_longer, attempts: 3
```

Comment:
Should we also retry on `ActiveRecord::RecordInvalid`?. When we `#save!` that could theoretically raise. I don't see how it could possibly end up invalid during this job, but it wouldn't hurt to retry.